### PR TITLE
fixes 3190 - generate env uses stdout instead of file

### PR DESCRIPTION
**Pull request by cdhanna**
#3190

### CODE REVIEW

File: cli/cli/Commands/Project/GenerateEnvFileCommand.cs [hunk 1]
```diff
@@ -36,30 +41,26 @@ public override void Configure()
...
-BEAM_INSTANCE_COUNT={args.instanceCount}
-REFRESH_TOKEN={refreshToken}
-";
-
-
+		var output = new GenerateEnvFileOutput
+		{
+			envVars = new List<EnvVarOutput>
+			{
+				EnvVarOutput.Create("SECRET", proj.secret),
```
Comment (gabrielbeamable) [inline cli/cli/Commands/Project/GenerateEnvFileCommand.cs:54]: Is there any possibility for these environment variables conflict with different ones in that process we are adding them? If so, would be nice to add a prefix or something to these.

**Review by gabrielbeamable — CHANGES_REQUESTED** [review]
Just a small concern that can be nothing

File: cli/cli/Commands/Project/GenerateEnvFileCommand.cs (hunk 1 above)
Comment (cdhanna) [reply cli/cli/Commands/Project/GenerateEnvFileCommand.cs:54]: fair enough feedback, but I think that would amount to a larger breaking change than we should do in a patch release. Do you mind if I make a ticket that gets on a 2.0 milestone?

Comment (gabrielbeamable) [reply cli/cli/Commands/Project/GenerateEnvFileCommand.cs:54]: seems good enough

**Review by gabrielbeamable — APPROVED** [review]

File: cli/cli/Commands/Project/GenerateEnvFileCommand.cs (hunk 1 above)
Comment (cdhanna) [reply cli/cli/Commands/Project/GenerateEnvFileCommand.cs:54]: https://github.com/beamable/BeamableProduct/issues/3231

**Pull request merged**